components/zoho_crm/sources/new-user/new-user.js

```diff
@@ -0,0 +1,86 @@
+const common = require("../common/timer-based/base");
+const userTypes = require("./user-types");
+
+module.exports = {
+  ...common,
+  key: "zoho_crm-new-user",
+  name: "New User",
+  description: "Emits an event each time a new user is created in Zoho CRM",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    userType: {
+      type: "string",
+      description: "The type of users to generate events for",
+      options: userTypes,
+    },
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 1, // 15 minutes
```

Comment:
Did you mean 15?

Reply:
Oops, yes I did. Nice catch!
I added this during local testing, I need to remove it.